Tasks/ShellScript/task.json

```diff
@@ -53,6 +53,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "disableAutoCwd",
+            "type": "boolean",
+            "label": "Disable Auto CWD",
+            "defaultValue": "false",
```

Comment:
Nevermind, it's a sticky situation... the checkbox only kicks in if !filePathSupplied. Visibility rule would just make matters worse.